### DISCUSSION TITLE
My Account got Suspended

### DISCUSSION THREAD

**Discussion by Shahbaz-Alee** [community/community#110686] [Other Feature Feedback, Questions, & Ideas] [answered]
### I GOT MY GITHUB SUSPENDED

General

### Body

My account was recently suspended for violating GitHub's terms and services. I reached out to GitHub Support yesterday, which was on a Friday. I would like to know the projected timeframe for the reinstatement of my account.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @Shahbazbalti ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

Regarding your topic: The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
